jujugui/static/gui/src/app/store/env/sandbox.js

```diff
@@ -1517,6 +1518,23 @@ YUI.add('juju-env-sandbox', function(Y) {
     },
 
     /**
+      Handle KeyManager.ListKeys messages.
+
+      @method handleKeyManagerListKeys
+      @param {Object} data The contents of the API arguments.
+      @param {Object} client The active ClientConnection.
+      @param {Object} state An instance of FakeBackend.
+    */
+    handleKeyManagerListKeys: function(data, client, state) {
+      client.receive({
+        'request-id': data['request-id'],
+        response: {results: [{
+          result: ['sandbox SSH key']
```

Comment:
I think the GUI will always request the full keys, and, even if it makes sense to have the real API generic enough to support both cases, I'd not propagate unrequired complexity to stuff like sandbox,  at least until it's proven to be required.